magclassic/automated_emails.py

```diff
@@ -1,19 +1,25 @@
 from magclassic import *
 
-# With "MAGFest Laboratories" as a new event, we find that the phrasing "Staffing again"
-# doesn't differentiate this event, so we remove it.
-AutomatedEmail.instances['Want to staff {}?'.format(c.EVENT_NAME)] = AutomatedEmail.instances.pop('Want to staff {} again?'.format(c.EVENT_NAME))
+# With "MAGFest Laboratories" as a new event, we find that the phrasing "Want to staff again?"
+# doesn't differentiate this event, so we change it to just plain "Want to staff?"
+if 'volunteer_again_inquiry' in AutomatedEmail.instances:
```

Comment:
I think I misunderstood the purpose of this code. I thought it was supposed to remove the " again" portion of the subject.